pipenv/utils/resolver.py

```diff
@@ -63,6 +63,16 @@
 err = rich.console.Console(stderr=True)
 
 
+class FakePipTempDirdirectory:
```

Comment:
I think having the name Fake in a class that is used in the actual production code is confusing, though I see why you added this.  Also it has `Dirdirectory`  Can we name it  `PipenvTempDirectoryWrapper` or something similar instead?

Reply:
Can we use  `QuasiPipTempDirectory`  instead? Wrapper is misleading because we aren't wrapping anything.

Reply:
Well we are wrapping the directory name 🤷   I am not sure it is Quasi either?

Reply:
Ah its fine -- better than Fake.